adr/0014-home-assistant-supervised.md

```diff
@@ -38,8 +39,9 @@ This installation method can easily be broken if one manages the operating syste
 - The operating system is dedicated to running Home Assistant Supervised.
 - All system dependencies are installed according to the manual.
 - No additional software, outside of the Home Assistant ecosystem, is installed.
-- Docker needs to be correctly configured to use overlayfs2 storage, journald as the logging driver, and cgroup v1.
+- Docker needs to be correctly configured to use overlayfs2 storage, journald as the logging driver with Container name as Tag, and cgroup v1.
```

Comment:
>  with Container name as Tag

It needs to be in a specific tag right? Otherwise we won't be able to look up logs for particular addons/core/etc. Are we going with `CONTAINER_NAME` or did we want the name as the syslog identifier.

Reply:
I don't know, at least we should make sure people run it as same as on OS

Reply:
Oh I see, you don't specify the tag name. We're just telling them to do this:
https://github.com/home-assistant/operating-system/blob/a6445af71282045c2ea92f1dea8d2f9e518bd008/buildroot-external/rootfs-overlay/etc/docker/daemon.json#L6-L8

That works fine then